From e900fff68af76d51a59ac085b35ace76939bc007 Mon Sep 17 00:00:00 2001
Patch-Source: https://src.fedoraproject.org/rpms/dotnet3.1/blob/rawhide/f/core-setup-no-werror.patch
From: Omair Majid <omajid@redhat.com>
Date: Tue, 18 Jan 2022 21:45:52 -0500
Subject: [PATCH] Disable Werror
 
This is so late in the release cycle that fixing warnings is just not
really worth it. The general approach is to fix the issues in the
development branches and disable warnings in the older release branches.
---
 src/settings.cmake | 1 -
 1 file changed, 1 deletion(-)
 
diff --git a/src/settings.cmake b/src/settings.cmake
index ff1e04f9..5cc9b3ef 100644
--- a/src/settings.cmake
+++ b/src/settings.cmake
@@ -201,7 +201,6 @@ else()
         # compiling with -std=c++11.
         # add_compile_options(-Weverything)
     endif()
-    add_compile_options(-Werror)
     add_compile_options(-Wno-missing-field-initializers)
     add_compile_options(-Wno-unused-function)
     add_compile_options(-Wno-unused-local-typedef)
-- 
2.34.1
